src/pages/myNotes/note/note.jsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Box } from '@material-ui/core';
+import { cutSymbols } from '../../../utils';
+import useStyles from '../styled';
+
+const Note = ({ activeNote: { title, description, date, id }, notActive, changeText }) => {
+  const classes = useStyles();
+  return (
+    <Box
+      className={notActive === id ? classes.activeNote : classes.note}
+      onClick={() => {
+        changeText(title, description, date, id);
+      }}
+    >
+      <p className={classes.title}>{title}</p>
+      { description !== '' && <p>{cutSymbols(description)}</p>}
```

Comment:
Pretty tricky hack. Don't use empty string!!
Replace it with: `description && <p>{cutSymbols(description)}</p>`